test/sphere.jl

```diff
@@ -85,4 +85,26 @@ include("utils.jl")
         vexp = normalize(project(M, x, [0, 1, zeros(n - 1)...]))
         @test v ≈ π * vexp
     end
+
+    @testset "Complex Sphere" begin
+        M = Sphere(2,ℂ)
+        @test repr(M) == "Sphere(2; field = ℂ)"
+        @test typeof(get_embedding(M)) === Euclidean{Tuple{3},ℂ}
+        @test representation_size(M) == (3,)
+        p = [1.0, 1.0im, 1.0]
+        q = project(M,p)
+        @test is_manifold_point(M,q)
```

Comment:
I'd suggest adding a test for tangent vector projection on the complex sphere.

Reply:
Or, even better, call `test_manifold` for complex sphere and tensor sphere.

Reply:
I am not so sure, we are again at 51 minutes currently for Julia 1.0, which already kills travis from time to time, running a whole `test_manifold` might be quite long (ok and I was lazy to think of 3 points on both of the manifolds).

Reply:
Then maybe we can have just a test for tangent vector projection here.

Reply:
Ok, I'll hack in a few numbers.